src/+otp/+kortewegdevries/+presets/Canonical.m

```diff
@@ -0,0 +1,48 @@
+classdef Canonical < otp.kortewegdevries.KortewegdeVriesProblem
+    methods
+        function obj = Canonical(varargin)
+
+            Domain = [-10, 10];
+            Nx = 200;
+            Theta = 0;
+            Nu = -1;
+            Alpha = -3;
+            Rho = 0;
+            Uinit = @(x) 6*(sech(x).^2);
```

Comment:
all lowercase

Reply:
alpha is a predefined function. That forced me to change it to Alpha. To keep consistency, I capitalized everything else. I didnt like using alpha when it was predefined. If you stil lwant me to change it, I will

Reply:
yes, please change. You are not using the alpha function, and the change is local. We use greek letters for variables a lot and we want to be consistent. Alpha as the name of a variable and not the name of a class makes the code impossible to read.

Reply:
Done

Reply:
Unit to uinit and I'm happy.

Reply:
Done